composer.json

```diff
@@ -61,5 +62,15 @@
     "branch-alias": {
       "dev-master": "4.0-dev"
     }
-  }
+  },
+  "repositories": [
+      {
+          "type": "git",
+          "url": "https://github.com/haltuf/DoctrineCache.git"
+      },
+      {
+          "type": "git",
+          "url": "https://github.com/haltuf/Annotations.git"
```

Comment:
Kdyby/Annotations PR has been merged. After new version is released, we still have to get rid of these temporary dependencies (replace haltuf/DoctrineCache and haltuf/Annotations with kdyby/*)